src/backend/api/utils.py

```diff
@@ -185,10 +187,9 @@ async def get_azure_subnets_json(http_session: ClientSession) -> Optional[Dict]:
             response_text = await resp.text()
     except client_exceptions.ClientConnectorError as err:
         logger.error(
```

Comment:
May using `exception` method be better than `error` when catching exception?

Reply:
Done